ambry-store/src/test/java/com/github/ambry/store/BlobStoreTest.java

```diff
@@ -1535,6 +1549,13 @@ public void putIdCollisionTest() throws Exception {
     // Ensure that all new entries were added.
     missingKeysAfter.clear();
     assertEquals(missingKeysAfter, store.findMissingKeys(allMockIdList));
+
+    // 5. first one absent, second one colliding and cleaned up journal.
```

Comment:
nit: we can better word the comments here, something like // 5. first be present, second one colliding but the journal is cleaned up. BTW, in case 4, we already put allMockIdList.get(3) to the store, so the first one is not absent.